HW_1/task1.py

```diff
@@ -0,0 +1,17 @@
+# Найдите сумму цифр трехзначного числа
+# Пример:
+# 123 -> 6 (1 + 2 + 3)
+# 100 -> 1 (1 + 0 + 0)
+
+number = int(input('Введите трехзначное число: '))
+while number // 1000 > 0 :   
+    if number//1000 > 0:
+        print('Число не трехзначное')
+        number = int(input('Введите трехзначное число: '))
```

Comment:
Зачем тут это условие, если оно уже есть в цикле while?
```suggestion
    print('Число не трехзначное')
    number = int(input('Введите трехзначное число: '))
```